en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -162,7 +162,16 @@ For platforms operating in a fixed wing mode, wind speed state estimation requir
 For multi-rotors, fusion of [Drag Specific Forces](#mc_wind_estimation_using_drag) can be enabled and tuned to provide the required wind velocity state estimates.
 
 The EKF2 module models the error as a body fixed ellipsoid that specifies the fraction of dynamic pressure that is added to/subtracted from the barometric pressure - before it is converted to a height estimate.
-See the following parameter documentation for information on how to use this feature:
+
+A good tuning is obtained as follows:
+1. Fly once in [Position mode](../flight_modes/position_mc.md) repeatedly forwards/backwards/left/right/up/down between rest and maximum speed (best results are obtained when this testing is conducted in still conditions).
```

Comment:
@bresch This section appears to apply to either MC OR FW but the instruction is "forwards/backwards/left/right/up/down" is it:
- We have to fly our FW backwards.
- This just means move the sticks in particular directions
- This only applies to MC - for FW you don't need to do baro tuning or you get this from airspeed?
- Something else?